Source/Krypton Components/Krypton.Ribbon/Group Contents/KryptonRibbonGroupTrackBar.cs

```diff
@@ -438,7 +439,15 @@ public int Value
         public int SmallChange
         {
             get => TrackBar!.SmallChange;
-            set => TrackBar!.SmallChange = value;
+            set
+            {
+                if (TrackBar != null)
+                {
+                    TrackBar.SmallChange = value;
+                }
+
+                // TODO: Return a default value
```

Comment:
The `//TODO:`'s are in the wrong place, they should be in the `get` function
e.g.
`get => Trackbar?.SmallChange ?? 0;`